docs/providers/aws/events/alb.md

```diff
@@ -103,3 +103,27 @@ module.exports.hello = async (event, context, callback) => {
 ```
 
 The handler response object must use `multiValueHeaders` to set HTTP response headers, `headers` would be ignored.
+
+## Prepending a prefix to generated target group names
+
+By default, target group names are strings generated by hashing a combined string of the function name, alb's id and whether multi-value headers are enabled. This produced a fixed length, unique id for target groups.
+
+Set the `providers.alb.targetGroupPrefix` attribute to a string to prepend this string to all target group names so that they become semi-predictable. The overall length is still truncated to 32 because this is a hard limit for target group names on AWS.
+
+Because of AWS's hard limit, `providers.alb.targetGroupPrefix` will also be limited to 16 characters in length to give ample length to the hash to ensure a high degree of uniqueness.
```

Comment:
Assuming we agree, on what I proposed above, let's remove this sentence

Reply:
Removed